Gemfile

```diff
@@ -92,6 +97,8 @@ unless ENV['APPLIANCE']
     gem "capybara",         "~>2.1.0",  :require => false
     gem "factory_girl",     "~>4.5.0",  :require => false
     gem "shoulda-matchers", "~>1.0.0",  :require => false
+    gem "vcr",              "~>2.6",    :require => false
+    gem "webmock",                      :require => false
```

Comment:
:+1: ok, so it should have been in the Gemfile all along, and we were missing it. cool